staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -79,12 +79,15 @@ type Event struct {
 	metav1.TypeMeta
 	// ObjectMeta is included for interoperability with API infrastructure.
 	// +optional
+	// DEPRECATED: Use StageTimestamp which supports micro second instead of ObjectMeta.CreateTimestamp
+	// and the rest of the object is not used.
 	metav1.ObjectMeta
 
 	// AuditLevel at which event was generated
 	Level Level
 
 	// Time the request reached the apiserver.
+	// DEPRECATED: Use RequestReceivedTimestamp which supports micro second instead.
```

Comment:
in the internal types we can potentially remove this field

Reply:
DONE.